ee-feature-pack/galleon-shared/src/main/resources/layers/standalone/web-server/layer-spec.xml

```diff
@@ -3,8 +3,8 @@
     <dependencies>
         <layer name="naming"/>
         <layer name="ee"/>
+        <layer name="ee-concurrency" optional="true"/>
```

Comment:
You can remove this. You have the 'ee' dep which in turn has the optional dep on ee-concurrency. So you don't need to declare it again.

Reply:
Thanks. I knew it looked funny, but couldn't quite figure out why :)